src/status_im/common/keychain/events.cljs

```diff
@@ -70,17 +70,13 @@
   {:db                        (assoc db :auth-method method)
    :keychain/save-auth-method [key-uid method]})
 
-(re-frame/reg-fx
- :keychain/get-auth-method
- (fn [[key-uid callback]]
-   (can-save-user-password?
```

Comment:
got rid of this check here, as we're doing it inside ` :biometric/check-if-available`